src/commands/server/deploy.ts

```diff
@@ -64,12 +64,20 @@ export default class Deploy extends Command {
       env: 'CHE_SERVER_BOOT_TIMEOUT'
     }),
     k8spodwaittimeout: string({
-      description: 'Waiting time for Pod Wait Timeout Kubernetes (in milliseconds)',
+      description: 'Waiting time for Pod scheduled condition (in milliseconds)',
+      default: '300000'
```

Comment:
Here and in the similar places like [this](https://github.com/che-incubator/chectl/pull/973/files#diff-1ad95a24a1800a51ca8b5449adc6c9d8c857a031f202a710aea4f5aaa17e86d2R50): please move the timeout into a constant to avoid different values